src/vue/components/panel-compact.vue

```diff
@@ -204,26 +268,40 @@
         <word-list-panel></word-list-panel>
       </div>
     </div>
-    <div class="alpheios-panel__notifications alpheios-text-small"
-         :class="{ 'alpheios-panel__notifications--important': $store.state.ui.notification.important }"
-         v-if="$store.state.ui.notification.visible" v-show="$store.state.ui.notification.important">
-            <span @click="$store.commit(`ui/resetNotification`)" class="alpheios-panel__notifications-close-btn">
-                <close-icon></close-icon>
+    <div
+        :class="{ 'alpheios-panel__notifications--important': $store.state.ui.notification.important }"
+        class="alpheios-panel__notifications alpheios-text-small"
+        v-show="$store.state.ui.notification.visible && $store.state.ui.notification.important"
+    >
+            <span
+                class="alpheios-panel__notifications-close-btn"
+                @click="$store.commit(`ui/resetNotification`)"
+            >
+                <close-icon/>
             </span>
       <span class="alpheios-panel__notifications-text" v-html="$store.state.ui.notification.text"></span>
-      <setting :classes="['alpheios-panel__notifications--lang-switcher alpheios-text-smaller']"
-               :data="settings.contentOptions.items.preferredLanguage"
-               @change="contentOptionChanged"
-               v-show="$store.state.ui.notification.showLanguageSwitcher"></setting>
+      <setting
+          v-show="$store.state.ui.notification.showLanguageSwitcher"
+          :classes="['alpheios-panel__notifications--lang-switcher alpheios-text-smaller']"
+          :data="settings.contentOptions.items.preferredLanguage"
+          @change="contentOptionChanged"
+      />
     </div>
-    <div class="alpheios-panel__notifications-auth alpheios-panel__notifications--important"
-         :data-count="$store.state.auth.notification.count"
-         v-if="$store.state.auth.notification.text" v-show="$store.state.auth.notification.count === 1 || $store.state.auth.notification.count % 10 == 0">
-         <span @click="$store.commit(`auth/resetNotification`)" class="alpheios-panel__notifications-close-btn">
-            <close-icon></close-icon>
+    <div
+        v-show="showLoginNotification"
+        class="alpheios-panel__notifications-auth alpheios-panel__notifications--important"
+        :data-count="$store.state.auth.notification.count"
+    >
+         <span
+             class="alpheios-panel__notifications-close-btn"
+             @click="$store.commit(`auth/resetNotification`)"
+         >
+            <close-icon/>
          </span>
-         <span v-html="l10n.getMsg($store.state.auth.notification.text)"></span>
-         <login v-show="$store.state.auth.notification.showLogin"></login>
+         <span
+             v-html="l10n.getMsg($store.state.auth.notification.text)"
+         />
+         <login/>
```

Comment:
the login element here actually did intentionally have separate condition than the notification, at least for now because the login button only shows for the webextension and not the embedded library. But I think that needs to be fixed further anyway per #388 so I am not sure if this is worth putting back or not.

Reply:
I've added a `TODO` as a reminder that we'll need to work on that